app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ReservationsController < ApplicationController
+  def new
+    #@reservation = Reservation.new
+    #@cart.data.each do |item_id, quantity|
```

Comment:
Incorrect indentation detected (column 4 instead of 6).<br>Missing space after #.